ci/benchmark_ai.sh

```diff
@@ -0,0 +1,149 @@
+    unzip -n JMeterPlugins-WebDriver-1.4.0.zip -d $HOME/apache-jmeter-4.0
+    cp ./lib/jmeter-plugins-cmn-jmeter-0.4.jar $HOME/apache-jmeter-4.0/lib/
+    cp ./lib/ext/* $HOME/apache-jmeter-4.0/lib/ext
+fi
+# end of jmeter installation
+
+echo "setting up conf"
+#Change conf files for running resnet
+cd /
+if [ -d "/mxnet-model-server" ]; then
+    git clone https://github.com/awslabs/mxnet-model-server.git
+fi
+sed -i -e 's/squeezenet=https:\/\/s3.amazonaws.com\/model-server\/models\/squeezenet_v1.1\/squeezenet_v1.1.model/resnet-18=https:\/\/s3.amazonaws.com\/model-server\/models\/resnet-18\/resnet-18.model /g'\
+ /mxnet-model-server/docker/mms_app_cpu.conf
+sed -i -e 's/squeezenet=https:\/\/s3.amazonaws.com\/model-server\/models\/squeezenet_v1.1\/squeezenet_v1.1.model/resnet-18=https:\/\/s3.amazonaws.com\/model-server\/models\/resnet-18\/resnet-18.model /g'\
+ /mxnet-model-server/docker/mms_app_gpu.conf
+sed -i -e 's/\/usr\/local\/Cellar\/jmeter\/3.3\/libexec\/lib\/ext\/CMDRunner.jar/\/home\/ubuntu\/apache-jmeter-4.0\/lib\/ext\/CMDRunner.jar /g' /mxnet-model-server/load-test/run_load_test.sh
+
+
+file="$HOME/log_results"
+
+if [ -f $file ] ; then
+    rm $file
+fi
+
+#start the container test
+for workers in 50 100
+do
+    for requests in 25 50 100
+    do
+        echo "*************************************** Workers $workers and reqs $requests   ****************************************************"
+        if [ "$use_gpu" == "0" ]; then
+            docker run  --name mms -v /mxnet-model-server:/mxnet-model-server  -itd -p 80:8080 awsdeeplearningteam/mms_cpu
+            docker exec mms pip install -U -e /mxnet-model-server/.
+            docker exec mms mxnet-model-server start --mms-config /mxnet-model-server/docker/mms_app_cpu.conf >& /dev/null &
+            sleep 30
+        elif [ "$use_gpu" == "1" ]; then
+            nvidia-docker run --name mms -v /mxnet-model-server:/mxnet-model-server -itd -p 80:8080 awsdeeplearningteam/mms_gpu
+            nvidia-docker exec mms pip install -U -e /mxnet-model-server/.
+            nvidia-docker exec mms pip uninstall --yes mxnet-cu90mkl
+            nvidia-docker exec mms pip uninstall --yes mxnet
+            nvidia-docker exec mms pip install mxnet-cu90mkl
+            nvidia-docker exec mms mxnet-model-server start --mms-config /mxnet-model-server/docker/mms_app_gpu.conf >& /dev/null &
+            sleep 90
+        fi
+
+        cd /mxnet-model-server/load-test
+        export JMETER_HOME=$HOME/apache-jmeter-4.0
+        export PATH=$JMETER_HOME/bin:$PATH
+        ./run_load_test.sh -i 127.0.0.1 -c $workers -n $requests -f report.csv -p 80
+        echo "done report generation"
+        OLDIFS= $IFS
+        IFS=","
+        read -r sampler_label aggregate_report_count average aggregate_report_median aggregate_report_90_line aggregate_report_95_line aggregate_report_99_line aggregate_report_min aggregate_report_max a\
```

Comment:
Move this parsing of log file to a seperate function, maybe parse_and_publish_metrics(). Take report.csv as a global variable, so that it can be referenced in the parse_and_publish_metrics()